### DISCUSSION TITLE
local secondary index

### DISCUSSION THREAD

**Discussion by romeosunio** [sensedeep/dynamodb-onetable#432] [Q&A] [answered]
I can only see an example for gsi.
```gs1: {hash: 'gs1pk', sort: 'gs1sk', project: ['gs1pk', 'gs1sk']},```

I assume that it is the same for lsi?  How to set `PROJECTION: ALL` option? Do we have to specify all attributes? An example would be nice. 

Thank you~

**Comment by mobsense** [comment, marked as answer]
Check the doc at:

https://doc.onetable.io

<img width="696" alt="image" src="https://user-images.githubusercontent.com/168091/210728383-9042c708-7ace-4f51-a84b-7a922a08a0d9.png">
